docs/src/development/proposals/MEP15/index.md

```diff
@@ -0,0 +1,168 @@
+
+From here everything should work the same as before but through remote accessing the BMC API.
+
+## New metalctl commands
+
+List BMCs:
+
+```
+metalctl bmc ls
+MAC                 IP         VENDOR       DRIVER     MACHINE ID
+27:53:57:51:6b:c8   10.0.0.8
+27:53:57:51:6b:c9   10.0.0.9
+92:33:b8:0e:df:8f   10.0.0.1   Supermicro   Redfish    37c43c25-69fe-4f88-b69d-4e71dc4070d0
+b3:74:fc:50:76:b6   10.0.0.4   Supermicro   Redfish    4bdf5c1b-3f7d-47df-84dd-05acb6e0718d
+56:62:97:4e:1f:1f   10.0.0.3   Dell         iDrac      995119fd-ec18-4cd7-8ca0-a9e1c2f70624
+```
+
+Describe a BMC:
+
+```
+metalctl bmc describe 92:33:b8:0e:df:8f
+---
+mac: 92:33:b8:0e:df:8f
+address: 10.0.0.1
+vendor: Supermicro
+protocol: Redfish
+machine_id: 37c43c25-69fe-4f88-b69d-4e71dc4070d0
+created_at: "2024-11-19T11:15:53.760Z"
+changed_at: "2024-11-19T11:18:53.760Z"
+bios:
+  date: 12/31/2021
+  vendor: American Megatrends Inc.
+  version: "3.6"
+board:
+  board_mfg: Supermicro
+  board_part_number: X11DPT-B
+  chassis_part_number: CSE-217BHQ+...
+  chassis_part_serial: C217BAK18P...
+  product_manufacturer: Supermicro
+  product_part_number: SYS-2029BT-HNR
+  product_serial: E262335X2304003C
+bmc:
+  version: "1.74"
+ipmi:
+  interface: lanplus
+  port: 623
+  password: abc
+  user: metal
+redfish:
+  password: abc
+  user: metal
+powermetric:
+  averageconsumedwatts: 70
+  intervalinmin: 5
+  maxconsumedwatts: 70
+  minconsumedwatts: 70
+  powerstate: "ON"
+  powersupplies:
+  - status:
+      health: Critical
+      state: Enabled
+  - status:
+      health: OK
+      state: Enabled
+ledstate:
+  description: ""
+  value: LED-OFF
+```
+
+Additional commands:
+
+```
+# establish initial access without metal-hammer
+metalctl bmc create-ipmi-user 92:33:b8:0e:df:8f --ipmi-role privileged --ipmi-password 123!
+# set preferred protocol
+metalctl bmc update 92:33:b8:0e:df:8f --preferred-protocol IPMI
+# enforce using Redfish implementation for this specific BMC
+metalctl bmc update 92:33:b8:0e:df:8f --preferred-protocol Redfish --redfish-user afish --redfish-password 123!
+```
+
+## Feature Deprecation
+
+In order to simplify the new implementation, we can deprecate some features.
+
+### Firmware Update Functionality
+
+This feature will be dropped because it was not completely worked out at the point of implementation. It also seems like nobody is actively using it. This brings so many challenges that we should create another MEP in order to bring it back when it's required.
```

Comment:
We could probably drop BIOS update and keep BMC Update functionality because the later does not require a system power reset.

Reply:
But how often did we really use it? :sweat_smile: 
I think the current implementation is kind of dangerous because it is not prevented from us that multiple update requests are issued in parallel against a single BMC. The operator does not even know if an update is currently in progress or not.

Reply:
Yes,  but this is a implementation flaw which must be addressed